src/decoder.rs

```diff
@@ -1018,9 +512,7 @@ impl Expr {
                     for v in values {
                         let arg = Value::Tuple(vec![Value::Seq(vs), v]);
                         // TODO can we avoid cloning arg here?
```

Comment:
Can remove this comment now that arg is not cloned 🤓

Reply:
Indeed, fixing it now.